recipes/bgAdmin.tsx

```diff
@@ -17,9 +17,9 @@ const BGCharmEntrySchema = {
     integration: { type: "string" },
     createdAt: { type: "number" },
     updatedAt: { type: "number" },
-    disabledAt: { type: "number" },
-    lastRun: { type: "number" },
-    status: { type: "string" },
+    disabledAt: { type: "number", default: 0 },
+    lastRun: { type: "number", default: 0 },
+    status: { type: "string", default: "" },
```

Comment:
Maybe worthwhile leaving a comment that this schema should (must?) match `CharmEntrySchema` in `@commontools/utils/updaters`

Probably a broader question, how does the worker code (via `@commontools/utils/updaters`) enforce/reject/ignore/handle data on read with a different written schema (e.g. `BGCharmEntrySchema` here)?

Reply:
good call - added the note

w.r.t. the broader question - as long as the schemas are additive the new defaults are loaded.

but I think we have to be careful - as I don't think anything stops us from breaking things with schema changes that aren't backwards compatible